### PR TITLE
extending the sdk nightly build with extra end to end test

### PR DESCRIPTION
**Pull request by wissamg**
## Component
- [X] Build
## Module
- [X] demisto-sdk

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
extending the sdk nightly build with extra end to end test

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .gitlab/ci/sdk-nightly.yml [hunk 1]
```diff
@@ -144,10 +158,73 @@ demisto-sdk-nightly:run-commands-against-instance:
     - section_end "Run Lint"
     - *change-file-ids
     - *upload-entities-to-cortex-xsoar
-    - section_start "Destroy instances"
-    - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
-    - section_end "Destroy instances"
+    # - section_start "Destroy instances"
+    # - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+    # - section_end "Destroy instances"
```
Comment (dorschw) [inline .gitlab/ci/sdk-nightly.yml:163]: is this commented-out on purpose?

File: .gitlab/ci/sdk-nightly.yml [hunk 2]
```diff
@@ -144,10 +158,73 @@ demisto-sdk-nightly:run-commands-against-instance:
...
+    # - demisto-sdk validate -i Packs/HelloWorld/
+    # - section_end "Run Validate"
+    # - section_start "Run Lint" --collapsed
+    # - demisto-sdk lint -i Packs/HelloWorld/
+    # - section_end "Run Lint"
+    # - *change-file-ids
+    # - *upload-entities-to-cortex-xsoar
+    # - section_start "Destroy instances"
+    # - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+    # - section_end "Destroy instances"
```
Comment (dorschw) [inline .gitlab/ci/sdk-nightly.yml:208]: commented out on purpose?

File: TestData/DemistoSDKE2E/demisto_sdk_e2e_test_integration/Pipfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "==5.0.1"
```
Comment (dorschw) [inline TestData/DemistoSDKE2E/demisto_sdk_e2e_test_integration/Pipfile:8]: why do we use a locked version here?

File: .gitlab/ci/sdk-nightly.yml (hunk 2 above)
Comment (wissamg) [reply .gitlab/ci/sdk-nightly.yml:208]: yes

Comment (dorschw) [reply .gitlab/ci/sdk-nightly.yml:208]: why? is it intended to stay commented out when merging?

Comment (wissamg) [reply .gitlab/ci/sdk-nightly.yml:208]: No No

**Pull request closed without merging**